.github/workflows/build.yml

```diff
@@ -0,0 +1,71 @@
+on:
+  pull_request:
+    branches: ["main"]
+  push:
+    branches: ["main"]
+  workflow_call:
+
+
+# In cases of concurrent workflows running (consecutive pushes to PR)
+# leave the latest workflow and cancel the other (older) workflows
+# See https://docs.github.com/en/actions/using-jobs/using-concurrency
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  actions: write
+  contents: read
+
+
+name: UI Bundle CI
+
+
+jobs: 
+  ui_bundle_skip:
```

Comment:
Probably don't need the skip since there aren't path filters within the bundle, etc. I also don't suspect we'll be changing thsi often.

Reply:
I thought of that but then I thought what's the harm :) 
I can remove it if you feel strongly about it

Reply:
I'm fine either way. I do think it adds a good deal of logical complexity (all the conditionals and stuff) without providing any benefit, so would be inclined to omit it until it was needed... but I don't have a strong opinion that direction.